### ISSUE TITLE
ExpandableGroup - List<Group> children 

### ISSUE THREAD

**Issue by shanibilu** [lisawray/groupie#112]
Is there a way to get the number of children?
The method "getGroupCount" only let me count the children when its expanded and the 'children' filed is private.

**Comment by lisawray** [comment]
Hm, I'm not sure what you're using that for.  The closest functionality in the library is `NestedGroup.getItemCount()`, which is protected.  That returns all the child items of the ExpandableGroup, including all the child items of the parent (expand toggle) group.  

OTOH, you could make an ExpandableGroup subclass which implements `getChildCount` and calls `getItemCount` inside, subtracting the children of the parent, which you'd have to catch a reference to in the constructor, as it is private.

tl;dr: I'm happy to consider modifying this structure, but I'm wondering what exactly you're trying to achieve here?

**Comment by R-Masoumi** [comment]
I realize this is a bit old but I have a scenario where this could be useful. when adding or removing groups dynamically when the number of sub groups reach zero you may want to remove the expandable item entirely, which is exactly the situation I find myself in.

**Comment by lisawray** [comment]
@R-Masoumi that's a good use case.  I will add this as a feature request.  I think that's a valid behavior to build into the expandable group itself as well
